src/Files.App/Helpers/FileOperationsHelpers.cs

```diff
@@ -546,33 +546,50 @@ public static void TryCancelOperation(string operationId)
 			}
 		}
 
-		public static Task<(string, ShellLinkItem)> ParseLinkAsync(string linkPath)
+		public static async Task<ShellLinkItem?> ParseLinkAsync(string linkPath)
 		{
+			if (string.IsNullOrEmpty(linkPath))
```

Comment:
Improvement: reduce code duplication. Match function to the one in [Win32Shell.cs.